metricsbp/baseplate_hooks.go

```diff
@@ -35,7 +34,10 @@ type SpanHook struct {
 }
 
 func newSpanHook(prefix string, metrics Statsd, span *tracing.Span) SpanHook {
```

Comment:
Oh so originally `BaseplateHook.Prefix` is supposed to be whatever we set in `metricsbp.Statsd` (via `StatsdConfig.Prefix`)? In that case we definitely don't need that in `BaseplateHook`.

Now do we still need prefix for this function? Is it still used anywhere?

Reply:
We do still use it, but I'm not sure we actually want to, I'm going to remove it.